### PR TITLE
New check-docker-deploy template to generalize CI/CD process

### PR DESCRIPTION
**Pull request by lluiscampos**
Inspired in current CI/CD from Mender Docs Site(s) repositories, and
meant to be reused across repos in the mendersoftware organization.

### CODE REVIEW

**Comment by lluiscampos** [comment]
It generalizes the logic from https://github.com/mendersoftware/mender-docs-site/blob/master/.gitlab-ci.yml#L24

**Comment by lluiscampos** [comment]
I verified it with some modifications in the pipeline and using a `dummy` branch. You can see the result here: https://github.com/mendersoftware/sre-tools/commits/dummy

**Review by tranchitella — APPROVED** [review]

File: .gitlab-ci-check-docker-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  - publish
+  - sync
+
+include:
+- project: 'Northern.tech/Mender/mendertesting'
+  file: '.gitlab-ci-check-docker-build.yml'
+
+variables:
+  INFRA_REPOSITORY: "git@github.com:mendersoftware/sre-tools.git"
+  INFRA_BRANCH: "master"
```
Comment (kacf) [inline .gitlab-ci-check-docker-deploy.yml:50]: Should probably include `DEBIAN_FRONTEND: noninteractive`.

Comment (lluiscampos) [reply .gitlab-ci-check-docker-deploy.yml:50]: :+1:

**Pull request merged**